server/src/main/java/io/envoyproxy/controlplane/server/limits/StreamLimiter.java

```diff
@@ -0,0 +1,85 @@
+package io.envoyproxy.controlplane.server.limits;
+
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCall.Listener;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+import io.grpc.Status;
+import java.util.concurrent.atomic.AtomicLong;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class StreamLimiter {
```

Comment:
Given that this is just a generic gRPC stream limiting interceptor, does it make sense to commit this to this repo? Seems like if it doesn't actually tie in with the control plane code itself it might just add to the maintenance burden without a clear benefit?

Maybe @joeyb has thoughts here?

Reply:
+1. Related: https://github.com/Netflix/concurrency-limits/tree/master/concurrency-limits-grpc

Reply:
I'm not feeling extremely strong about it, but it seems more feature complete if it resides in this repository. To provide a full example such as the one in `TestLimits` would require importing the external dependency from Netflix in the test package.